### ISSUE TITLE
email verification not working

### ISSUE THREAD

**Issue by Ardin516** [codewars/codewars.com#2250]
When trying to verify my codewars account Ardin17 email Ardin516@protonmail.com, i get the email but it refuses to verify and gives me a 403 error after being redirected to the codewars site.

**Comment by kazk** [comment]
Some users have reported the same issue using some browsers on Android. Can you provide your browser information (the one that opened the confirmation link)?

**Comment by Ardin516** [comment]
yes sorry, i missed the reply, the browser it was sent from was chrome, the browser i tried to verify it from was firefox.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by NALIN2022** [comment]
I have the same problem for confirm email , it just show me 403 error said that I have no permission.

**Comment by kazk** [comment]
Make sure to open the link with the browser you're signed in with. The confirmation link doesn't log you in because that would allow someone to take over your account.

It should be redirecting to the sign-in page instead of showing 403 error, so I'll look into that.
